keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -161,11 +203,11 @@ proposed in this KEP.
 When a new Pod is created, Scheduler is responsible for selecting a suitable
 Node that accommodates the Pod.
 
-For a newly created Pod, Spec.Containers[i].ResourcesAllocated must match
-Spec.Containers[i].Resources.Requests. When Kubelet admits a new Pod, values in
-Spec.Containers[i].ResourcesAllocated are used to determine if there is enough
-room to admit the Pod. Kubelet does not set Pod's ResourcesAllocated after
-admitting a new Pod.
+For a newly created Pod, the apiserver will set the `ResourcesAllocated` field
```

Comment:
@vinaykul I am not sure why this particular nuance is important - is it just to avoid kubelet needing to write to the pod immediately?  Since we need pod IPs for things like Endpoints - is it really a savings?

Reply:
Yes, but it is less important now. It made sense when ResourcesAllocated was in PodSpec as we would avoid one write-back to PodSpec just to set it. Now, this is more like initializing Phase or QOSClass.

Reply:
If you are happy with this PR's overall changes to the KEP, approve it and then you can update it to reflect a new thinking on this point?